classes/class06/chaz_homework/models/twoteModel.js

```diff
@@ -0,0 +1,8 @@
+var mongoose = require('mongoose');
+
+var twoteSchema = mongoose.Schema({
+	text: String,
+	user: String
```

Comment:
Would be nice to do this as a connection to the User model, make it more extensible